scalafix/input/src/main/scala/fix/CompanionSrc.scala

```diff
@@ -3,40 +3,40 @@ rule = "scala:fix.CrossCompat"
  */
 package fix
 
-import scala.collection.immutable
-import scala.collection.mutable
+import scala.{collection => c}
```

Comment:
@olafurpg

there is a regression in replaceSymbol.

Reply:
can you open a ticket? I'm not sure exactly what the regression is